docs/src/main/sphinx/connector/accumulo.rst

```diff
@@ -24,6 +24,15 @@ JAR file to Accumulo's ``lib/ext`` directory on each TabletServer node.
 
     # TabletServer should pick up new JAR files in ext directory, but may require restart
 
+Requirements
+------------
+
+To connect to Accumulo, you need:
+
+* Accumulo version 1.7.4 or higher.
```

Comment:
Doesn't work with Accumulo 2.x IIRC.

Reply:
Correct, doesn't work with 2.x. Relates to https://github.com/trinodb/trino/issues/7297

Reply:
Maybe reword to 

```
Accumulo versions 1.x starting with 1.7.4. Versions 2.x are not supported.
```